### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by SpectralKH** [SpectralKH/Sylver#14]
:)

**Comment by forddyce** [comment]
What is this? A smiley? Has anything to do with the web?

**Comment by forddyce** [comment]
Oh, I might be updating something tomorrow, for home page.

**Comment by SpectralKH** [comment]
@forddyce If there's an "Incomplete" label, it's unfinished. And, cool, I'm excited to see what you got!

**Comment by forddyce** [comment]
Pushed something, doesn't seem to be working live though, can you explain
how those github pages work?

**Comment by SpectralKH** [comment]
@forddyce I gave you the wrong address, it's actually http://spectralkh.github.io/Sylver/. If that doesn't work, simply google GitHub Pages.

**Comment by forddyce** [comment]
it only works with html file? wow, my files are in php, gonna change all
first

**Comment by SpectralKH** [comment]
@forddyce :/ Ahh, I see. It seems like not everything is supported here, I'm not exactly sure what. Read something about databases as well. I'll do some research.

**Comment by forddyce** [comment]
well, if it does not support the technologies needed, will be very hard for
me to provide you a preview, and html is very limited to what you need
later on

**Comment by SpectralKH** [comment]
I'm aware. I'll definately get a solution.

**Comment by SpectralKH** [comment]
@forddyce @bluckjigar Anyone got a suggestion for a host to use? Looking for one that can suit our simple needs right now, thus a low price would be nice.

**Comment by forddyce** [comment]
I will see if my friend will allow me to host some files on his hosting. He still owes me. He is asleep now so he will reply when he wakes up. I will contact you if he agrees.

**Comment by SpectralKH** [comment]
@forddyce Did you get tell him yet?

**Comment by forddyce** [comment]
yes, he agreed, but we need the domain name though

**Comment by SpectralKH** [comment]
@forddyce I'll get one. Any more info about the hosting I could get?

**Comment by forddyce** [comment]
I think you should get the hosting too instead of just the domain name, my
employer's hosting apparently doesn't have the latest requirement (his php
is still 5.3), and upgrading them might affect his client's site.

Here are the requirements:

   - PHP >= 5.4
   - Mcrypt PHP Extension
   - OpenSSL PHP Extension
   - Mbstring PHP Extension
   - Tokenizer PHP Extension

**Comment by SpectralKH** [comment]
@forddyce Then I'm thinking about Site5. Also checking if Arvixe is supporting it.

**Comment by SpectralKH** [comment]
@forddyce Do we need JSP? If not, I'm purchasing from Arvixe.

**Comment by forddyce** [comment]
no, no,we don't, not yet at least, i can't think of the jsp usage now
